api/.env.defaults

```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```

Comment:
@johnbenjamin-hmcts - default value has changed to `demo`... is this intended?

Reply:
@Daniel-Lam - it was done in August 2022 and I am not sure. https://github.com/hmcts/rpx-xui-webapp/commit/ed5950ea5771743f8605b83cac2166a458cf607a#diff-b22918fe0c9196fb971c7628addde1108687383c4e381d6d5c8358b6e12bcef9

It is `development` in the master branch, I will change it to match the master.

Reply:
thanks, @johnbenjamin-hmcts